common/rc

```diff
@@ -4538,7 +4546,7 @@ init_rc()
 
 	# Sanity check that TEST partition is not mounted at another mount point
 	# or as another fs type
-	_check_mounted_on TEST_DEV $TEST_DEV TEST_DIR $TEST_DIR $FSTYP || exit 1
```

Comment:
Why is this changed needed? This should go upstream?

Reply:
And what is if FUSE_SUBTYPE is not set?

Reply:
Sorry, I still cannot follow you. Why does plain fuse work? It should be an issue without redfs? To me this looks like a non-redfs related change that  needs it is own commit message with an explanation why it is needed.

Reply:
plain fuse would not work if -o subtype option is provided in /sbin/mount.fuse.passthrough_hp script. As we are running xfstestuite on passthrough_hp and passthrough_hp requires separate mount script rather than passthrough_ll one, we need subtype option so that mount would uses correct script to run the underlying binary.